host_core/lib/host_core/actors/actor_module.ex

```diff
@@ -183,7 +183,8 @@ defmodule HostCore.Actors.ActorModule do
       instance_id = Agent.get(agent, fn content -> content.instance_id end)
       Tracer.set_attribute("instance_id", instance_id)
 
-      {:ok, module} = Wasmex.Module.compile(bytes)
+      {:ok, store} = Wasmex.Store.new()
```

Comment:
This could be the diff confusing me, but doesn't this create a "non-WASI" store that won't supply the right wasi imports to the module? Wasn't the plan to always create a wasi store because it would be harmless to do so for a non-WASI actor?

Reply:
Good catch. We swapped this in one place yesterday but not the other. TY!

Reply:
Hah! Was just checking against my senility